common/supplemental/metaZones.xml

```diff
@@ -1892,6 +1892,167 @@ For terms of use, see http://www.unicode.org/copyright.html
 			<mapZone other="Yerevan" territory="001" type="Asia/Yerevan"/>
 			<mapZone other="Yukon" territory="001" type="America/Whitehorse"/>
 		</mapTimezones>
+
+		<shortIds>
+            <shortId name="acre" description="Metazone" alias="Acre" since="40"/>
```

Comment:
description="Metazone" is redundant. This data is now under <metaZones><shortId>, so it's metazone ID obviously. We could add meaningful description later, for example "Time used by Acre state in Brazil".

Reply:
Because inception of this data is CLDR 40, we could default to since="40" and use since="xx" only for future additions.

Reply:
The display name of the metazone should come from CLDR.

Do we want this supplemental item to be "meritable facts about this metazone"? or is it strictly "short aliases"? If it's the former, we should have a different structure and the element should be `<metazoneInfo>` or such. 

We don't currently have any chunk of data that exists one time for each metazone.

Reply:
I actually prefer to have a complete list of metazones - with long/short IDs.
At this moment, if you want to get all valid metazone IDs, you have to walk through the mapping data and collect all of unique IDs.

Reply:
@yumaoka we can do that, but that's a different type of structure. right now `metazoneInfo` AND `mapTimezones` each have mappings to metazones. I'd assume we don't want to actually change those existing items?

If we want an element that is the canonical list of metazones, I'd name it so:


```xml
<metazoneIds>
  <metazoneId type="Europe_Western" alias="euwe" since="41"/> <!-- assuming a non-default 'since' -->
</metazoneIds>
```

This structure and naming would allow for other metadata on a per-metazone basis.

Reply:
Maybe something like below?

```
<metaZones>
  <metazoneList>
    <metazone type="Acre" short="acre" description="Time for Acre, Brazil"/>
    ....
  </metazoneList>
  <metazoneInfo>
    <!-- tz/mz mapping data -->
    ...
  </metazoneInfo>
</metaZoens>
```

If we did not have <metaZones> element, we would want to use `<metazones>` instead of `<metazoneList>` above. The element <metazone> describes the definition of each metazone - comes with long id and short id.